compiler/src/main/java/com/juul/pommel/compiler/internal/Utils.kt

```diff
@@ -28,6 +37,12 @@ internal val fragmentComponent: ClassName = ClassName.get("dagger.hilt.android.c
 internal val serviceComponent: ClassName = ClassName.get("dagger.hilt.android.components", "ServiceComponent")
 internal val viewComponent: ClassName = ClassName.get("dagger.hilt.android.components", "ViewComponent")
 
+internal val MODULE = ClassName.get("dagger", "Module")
+internal val PROVIDES = ClassName.get("dagger", "Provides")
+internal val BINDS = ClassName.get("dagger", "Binds")
+internal val INSTALLIN = ClassName.get("dagger.hilt", "InstallIn")
+internal val GENERATED = ClassName.get("javax.annotation", "Generated")
```

Comment:
Format for declaring new `ClassName` doesn't match the format used for declaring the old ones.